positioning/styles.css

```diff
@@ -0,0 +1,139 @@
+/*using custom properties, we're defining where the z-index actually "is" */
+:root {
+  --z-1: 100;
+  --z-2: 200;
+  --z-3: 300;
+  --z-4: 400;
+  --z-5: 500;
+  --z-6: 600;
+  --z-7: 700;
+  --z-8: 800;
+  --z-9: 900;
+}
+
+body {
+  margin: 0;
+  font: 1rem / 1.5 sans-serif;
+}
+
+main {
+  /* border: 1px solid red; */
+  margin: 6rem auto;
+  width: 95%;
+  max-width: 75ch;
+  font-size: 1.125rem;
+}
+
+figure {
+  margin: 2rem 0;
+  padding: 0;
+  position: relative;
+  z-index: var(--z-1);
```

Comment:
You actually shouldn't need to define a  z-index here, since elements will automatically have a z-index of 0 by default your header with a z-index of 9 should always stay layered above it. And since your photo caption is positioned absolute it is also automatically stacked for you since the absolute position pulls the element out of the natural flow of the DOM.